.nvmrc

```
@@ -1 +1 @@
-v18.13.0
\ No newline at end of file
+v18.18.2
```

I think we can directly remove minor version
```suggestion
v18
```

---

Me personally, I like to set up environments as much predictable as possible in regards of [reproducibility](https://reproducible-builds.org/), that's why I would vote to define the exact version here.

But I would also be fine with @j0k3r's approach if this is the preferred way.